src/NVelocity/SupportClass.cs

```diff
@@ -17,7 +17,6 @@ namespace NVelocity
 	using System;
 	using System.Collections;
 	using System.Data;
-	using System.Data.OleDb;
```

Comment:
This was referenced and not used, unsure if .NET Core has an equiv but removing it didn't break the tests.